lab_2_retrieval_w_bm25/main.py

```diff
@@ -33,6 +44,17 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(tokens, list) or not isinstance(stopwords, list):
+        return None
+    if not all(isinstance(token, str) for token in tokens) or not all(isinstance(word, str) for word in stopwords):
+        return None
+    if tokens == [] or stopwords == []:
+        return None
+    new_tokens = []
+    for token in tokens:
```

Comment:
try to explore list comprehension. It will make your code better https://www.w3schools.com/python/python_lists_comprehension.asp

Reply:
done)